### PR TITLE
[DOCS] Add draft of puppeth getting started guide

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
If you're happy with the technical side of the content I'll add some images and a bit more polish.

Vagrant file is here - https://github.com/ChrisChinchilla/Puppeth-Vagrant can move it, but actually prerequisites were quite hard, so this seemed a good thing to provide.

### CODE REVIEW

**Comment by adamschmideg** [comment]
@karalabe says
- Vagrant makes it more complicated. Say you use
- Using only menu numbers is fragile, hard to maintain
- Expand on the linked article or just link to it

**Comment by karalabe** [comment]
So, the problem with this guide is that it's way too opaque. Writing "press 4, then 6, then 7, then you should see a table" is not really helpful. It is super brittle which will get out of date the first time we update puppeth. Also without actual intput/output details and screenshots, it's impossible to follow. I saw that you linked https://www.sitepoint.com/puppeth-introduction/ . I honestly think that that blog post is a very very nice resource, and maybe we should just take it as is (ask for permission) and then use that as a starting point and extend/polish it instead of rewriting from zero.

**Comment by ChrisChinchilla** [comment]
OK… @adamschmideg and @karalabe 

Vagrant is just an option for people to quickly and easily test things, not the way to do it in production. If people want to test, then asking them to install and setup everything themselves  starts to get a bit beyond a "getting started guide". Unless we fill in the gaps before this guide and have steps for getting all the other steps set up, and then say "now with those installed, here's how you do puppeth…"

The opaqueness is easy to sort. Screenshots were noted as coming too.

I can ask permissions to just reuse the blog post (I actually know the writer), but this somewhat depends on your thoughts on the above questions.

I'll wait and hear back on those questions first and go from their.

File: docs/_getting-started/creating-private-network-puppeth.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+A private Ethereum network is useful for dapp testing, privacy, starting a local network for a hackathon, and many other purposes. An Ethereum network is private if the nodes are not connected to a main network.
+
+Setting up a private network can be a complex task; you need to configure a genesis block, different node types, miners, monitoring, and more. To help get you started, `puppeth` is a CLI tool that leads you through the steps to create a private network, building it with Docker containers, and in creating a new Ethereum network down to the genesis, bootnodes, signers, ethstats, faucet, dashboard and more, without the hassle that it normally takes to configure all these services one by one. Puppeth uses ssh to connect into remote servers and builds its network components out of docker containers using docker-compose. The user is guided through the process via a command-line wizard that does the heavy lifting and topology configuration automatically behind the scenes.
+
+## Prerequisites
+
+Any node you want to connect to the private network has some prerequisites:
+
+- The ethereum toolchain
```
[the remark this answers is not in the document]
Comment (ChrisChinchilla) [reply docs/_getting-started/creating-private-network-puppeth.md]: I'll link, it's just the `ethereum` package available in many package managers

**Comment by adamschmideg** [comment]
We had an internal conversation on this. The simplest way to move forward would be to ask the author of the guide for permission to use his guide.

**Comment by ChrisChinchilla** [comment]
OK @adamschmideg I know him pretty well, so I'll ask. I think for good experience for Geth users I'll ask if they mind if we reproduce it in place, instead of linking to it. If not, I'll take another go at this, and make it even more different, but there's only so many directions you can take it anyway, so any tutorial will end up looking a little familiar, and even mine is not exactly the same.

**Comment by ChrisChinchilla** [comment]
Spoke with Bruno, he is fine for us to reference, or take heavy inspiration from his article. The best approach seems to be take what I already made, and update it to reflect the comments and make it more long-lasting and less "broken" by changes.

**Pull request closed without merging**